packages/wrangler/src/api/dev.ts

```diff
@@ -10,7 +10,7 @@ interface DevOptions {
 	env?: string;
 	ip?: string;
 	port?: number;
-	noBundle?: boolean;
```

Comment:
you changed this here, but not anywhere else it's read. intentional?

Reply:
Yeah, we don't actually use `args.noBundle`, it's `args.bundle`

Reply:
Incorrect typing